website/source/docs/guides/acl-replication.md

```diff
@@ -0,0 +1,244 @@
+
+```sh 
+$ consul acl token create description "replication token" -policy-name
+replication 
+AccessorID:   67d55dc1-b667-1835-42ab-64658d64a2ff 
+SecretID:     fc48e84d-3f4d-3646-4b6a-2bff7c4aaffb 
+Description:  replication token 
+Local:        false
+Create Time:  2019-05-09 18:34:23.288392523 +0000 UTC 
+Policies: 
+  240f1d01-6517-78d3-ec32-1d237f92ab58 - replication 
+```
+
+## Enable ACL Replication on the Secondary Datacenter
+
+Once you have configured the primary datacenter and created the replication
+token, you can setup the secondary datacenter. 
+
+### Configure the Servers 
+
+You will need to set the `primary_datacenter` parameter to the name of your
+primary datacenter, `enable_token_replication` to true, and `retry_join_wan` to
+the primary datacenters server's addresses. By configuring the retry join
+parameter for wan gossip, you are also setting up [basic
+federation](/consul/security-networking/datacenters). 
+
+```json 
+{ 
+"datacenter": "dc_secondary", 
+"retry_join_wan": ["<address_here>"],
+"primary_datacenter": "primary_dc", 
+"acl": { 
+	"enabled": true, 
+	"default_policy": "deny", 
+	"down_policy": "extend-cache", 
+	"enable_token_persistence": true,
+    "enable_token_replication": true, 
+    } 
+} 
+```
+
+Now you can start the agent.
+
+```sh 
+$ consul agent -config-file=server.json 
+``` 
+
+To complete any further
+actions, the servers will need agent tokens. You can create server specific
+tokens on your primary datacenter and then set the token on the server.  
+
+```sh
+$ consul acl set-agent-token <token here> 
+ACL token "agent" set successfully
+``` 
+-> Note, your initial bootstrap token can be used for the necessary
+privileges to complete any action on the secondary servers. 
+
+Repeat this process on all servers. 
+
+### Configure the Clients
+
+For the clients, you will need to set the `primary_datacenter` parameter to the
+name of your primary datacenter and `enable_token_replication` to true.
+
+```json 
+{ 
+"datacenter": "dc_secondary", 
+"primary_datacenter": "primary_dc",
+"acl": { 
+	"enabled": true, 
+	"default_policy": "deny", 
+	"down_policy": "extend-cache", 
+	"enable_token_persistence": true, 
+    "enable_token_replication": true, 
+    } 
+} 
+```
+
+Now you can start the agent.
+
+```sh 
+$ consul agent -config-file=server.json 
+``` 
+
+Repeat this process on all
+clients. 
+
+### Apply the Replication Token to the Servers
+
+Finally, apply the replication token to all the servers using the CLI. 
+
+```sh 
+$ consul acl set-agent-token replication <token> 
+ACL token "replication" set successfully 
+```
+
+Once token replication has been enabled, you will also be able to create
+datacenter local tokens.
+
+Note, the clients do not need the replication token. 
+
+## Check Replication 
+
+Now that you have set up ACL replication, you can use the [HTTP API] to check
+the configuration.
+
+```sh 
+$ curl http://localhost:8500/v1/acl/replication
```

Comment:
```suggestion
$ curl http://localhost:8500/v1/acl/replication?pretty
```

Adding the `?pretty` will cause our API to return prettyfied responses and should more closely match the output below.